### PR TITLE
refactor: sdk integration

### PR DESCRIPTION
**Pull request by NexPB**

### CODE REVIEW

File: assets/easy_points.js [hunk 1]
```diff
@@ -112,6 +109,63 @@ var EasyPointsCore = {
     },
   },
 
+  Locale: {
```
Comment (Tw33t3r) [inline assets/easy_points.js:112]: Flag: remove when tiers finshed

File: assets/easy_points.js [hunk 2]
```diff
@@ -380,43 +389,37 @@ var EasyPointsUI = {
   }
 };
 
+function updateAllLoyaltyTargets() {
```
Comment (Tw33t3r) [inline assets/easy_points.js:392]: Flag: tiers

File: assets/easy_points_integration.js [hunk 3]
```diff
@@ -591,16 +651,52 @@ var EasyPoints = {
...
+    if (cartItems.getSectionsToRender === undefined) return;
+    if (cartItems.getSectionInnerHTML === undefined) return;
+
+    var sections = cartItems.getSectionsToRender();
+
+    var templates =
+      sections.map((section) => section.section)
+        .join(',');
+
+    fetch(`${window.Shopify.routes.root}?sections=${templates}`)
```
Comment (Tw33t3r) [inline assets/easy_points_integration.js:672]: Can we push this to SDK and call from here?

File: assets/easy_points.js [hunk 4]
```diff
@@ -112,6 +109,63 @@ var EasyPointsCore = {
...
+        const { locale } =  EasyPointsData.shop;
+
+        return locale || defaultLocale;
+      }
+
+      return defaultLocale;
+    }
+  },
+
+  Currency: {
```
Comment (Tw33t3r) [inline assets/easy_points.js:126]: Flag: Tiers

File: assets/easy_points_integration.js (hunk 3 above)
Comment (NexPB) [reply assets/easy_points_integration.js:673]: https://github.com/SolarisJapan/EasyPoints/pull/964

**Pull request merged**